File: Data Structures & Algorithms/is-anagram/submission-0.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        mp = {}
        for i in s:
            mp[i] = mp.get(i,0)+1
        for i in t:
            if i not in mp:
                return False
            mp[i] -= 1
        for i in mp:
            if mp[i] != 0:
                return False
        return True
            
       
       
    

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
     int buy = 0;
     int sell = 1;
     int maxp = 0;
     for(sell = 1;sell<prices.length;sell++){
        if(prices[sell]>prices[buy]){
            int profit = prices[sell] - prices[buy];
            maxp = Math.max(maxp,profit);
        }else{
            buy = sell;
        }
        
     }
     return maxp;
    }
}
